_episodes/02-bisecting.md

```diff
@@ -79,7 +79,7 @@ Use `git bisect` to find the commit which broke the computation.
 ### How to find the first commit
 
 ```
-$ git log --oneline --reverse
+$ git log --oneline --reverse | head
```

Comment:
`git log --oneline | tail -1` - might be a good option if I understood what we are looking for with this

Reply:
Thanks! I will revise this.